Makefile

```diff
@@ -0,0 +1,34 @@
+COMPOSER = $(shell if composer --version > /dev/null 2> /dev/null; then \
+		echo 'composer';\
+	elif [ -f composer.phar ]; then \
+		echo 'php composer.phar';\
+	else \
+		echo 'error';\
+	fi)
+
+.PHONY: help
+help:
+	@echo "Please use \`make <target>\` where <target> is one of"
+	@echo "  all    to check and build the plugin"
+	@echo "  build  to build the plugin"
+	@echo "  check  to check \`composer\`"
+
+.PHONY: all
+all: check build
+
+.PHONY: check
+check:
+	@echo 'checking...'
+ifeq (${COMPOSER}, error)
+	$(error Please install composer.)
+else
+	@echo 'OK'
+endif
+
```

Comment:
We need to add a `make release` to build zip package which can be installed directly.